lib/modules/apostrophe-schemas/public/js/user.js

```diff
@@ -1040,6 +1039,41 @@ apos.define('apostrophe-schemas', {
       convert: convertString
     });
 
+    self.addFieldType({
+      name: 'colorpicker',
+      populate: function (data, name, $field, $el, field, callback) {
+        $field.val(data[name]);
+        apos.ui.enhanceColorpicker($field);
+        $field.next().text(data[name]);
+        $field.on('change', function() {
+          $field.next().text($field.val());
+        });
+        return setImmediate(callback);
+      },
+      convert: function (data, name, $field, $el, field, callback) {
+        data[name] = $field.val();
```

Comment:
Deal with required-ness. If that's not being picked up elsewhere somehow, you might test first.